src/main/java/AccountManager.java

```diff
@@ -0,0 +1,51 @@
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class AccountManager {
+
+    private HashMap<String, Person> users;
+
+    public AccountManager() {
+        users = new HashMap<>();
+    }
+
+    public boolean createAccount() {
+
+        AccountCreator creator = new AccountCreator();
+        Person user = new Person();
+
+        Scanner in = new Scanner (System.in);
+        System.out.println("What is your desired user name?");
+        String name = in.nextLine();
+        System.out.println("What is your password?");
+        String pass = in.nextLine();
+        System.out.println("What is the answer to your security question?");
+        String answer = in.nextLine();
+        boolean result = creator.makeAccount(user, name, pass, answer, this.users);
+        if (result) {
+            users.put(name, user); // adds the user to the hashmap of users
+        }
+        return result;
+    }
+
+    public boolean changePassword() {
+
+        AccountEditor editor = new AccountEditor();
+        Scanner in = new Scanner (System.in);
+
```

Comment:
same issue of interacting directly with user when i don't think it's supposed to -- could we instead make the username a parameter?

Reply:
I have changed the required values to parameters.